aip/general/0148.md

```diff
@@ -0,0 +1,139 @@
+---
+id: 148
+state: reviewing
+created: 2020-09-14
+placement:
+  category: fields
+  order: 90
+---
+
+# Standard fields
+
+Certain concepts are common throughout any corpus of APIs. In these situations,
+it is useful to have a standard field name that is used consistently to
+communicate that concept.
+
+## Guidance
+
+Standard fields **should** be used to describe their corresponding concept, and
+**should not** be used for any other purpose.
+
+### Resource names and IDs
+
+#### name
+
+Every resource **must** have a `string name` field, used for the resource name
+(AIP-122), which **should** be the first field in the resource.
+
+**Note:** The `_name` suffix **should not** be used to describe other types of
+names unless otherwise covered in this AIP.
+
+#### parent
+
+The `string parent` field refers to the resource name of the parent of a
+collection, and **should** be used in most `List` (AIP-132) and `Create`
+(AIP-133) requests.
+
+#### uid
+
+The `string uid` field refers to a system-assigned, unique identifier for a
+resource. When provided, this field **should** be a [UUID4][].
+[Declarative-friendly resources][] **should** include this field.
+
+<!-- prettier-ignore -->
+[uuid4]: https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)
+
+#### short_name
```

Comment:
I could use some clarification on the difference between a short-name and a user-specified ID. Are they the same? What is "within a namespace"?

Reply:
Actually this just needs to be removed. My mistake.